### PR TITLE
enhance navbar

### PR DESCRIPTION
**Pull request by KennethKnows**
Updated

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -72,64 +72,64 @@ const Navbar: React.FC<NavbarProps> = () => {
...
                             >
-                                CONTACTS
+                                CONTACTS 
                             </button>
                             <button
-                                className="text-white no-underline font-normal transition duration-300 hover:text-[#252F70]"
+                                className="text-primary no-underline font-medium transition duration-300 hover:text-[#252F70]"
                                 onClick={() => handleNavigation('/signup')}
                             >
-                                CREATE ACCOUNT
```
Comment (kynttt) [inline src/components/Navbar.tsx:102]: do not delete this tab.

Comment (kynttt) [reply src/components/Navbar.tsx:102]: try to click also the tabs if it is redirecting to its intended page.

**Review by kynttt — CHANGES_REQUESTED** [review]
check comments

File: src/components/Navbar.tsx (hunk 1 above)
Comment (KennethKnows) [reply src/components/Navbar.tsx:102]: resolve the issue already

**Review by kynttt — CHANGES_REQUESTED** [review]
Still issues persist. Try to navigate each tab, specifically "service" and "about us". Also, you added "career" tab, make sure to add it also on 'Responsive Navigation Menu' for smaller screens navbar.

**Comment by KennethKnows** [comment]
Added updates - about and service + add in responsive menu bar

**Comment by KennethKnows** [comment]
navbar update 07/24/2024

**Comment by kynttt** [comment]
merge w/ update

**Pull request merged**